Protocol.md

```diff
@@ -601,17 +613,21 @@ The message SHALL NOT be encrypted.
 
 After the 'client-hello' message has been sent (responder) or after the
 'server-hello' message has been received (initiator) the client MUST
-send this message to the server. The client MUST set the *your_cookie*
-field to the cookie the server has used in the nonce of the
-'server-hello' message. It SHALL also set the *subprotocols* field to
-the exact same `Array` of subprotocol strings it has provided to the
-WebSocket client implementation for subprotocol negotiation. If the
-user application requests to be pinged (see
-[RFC 6455 section 5.5.2](https://tools.ietf.org/html/rfc6455#section-5.5.2))
-in a specific interval, the client SHALL set the field *ping_interval*
-to the requested interval in seconds. Otherwise, *ping_interval* MUST
-be set to `0` indicating that no WebSocket *ping* messages SHOULD be
-sent.
+send this message to the server.
+
+* The client MUST set the *your_cookie* field to the cookie the server
+  has used in the nonce of the 'server-hello' message.
+* It SHALL also set the *subprotocols* field to the exact same `Array`
+  of subprotocol strings it has provided to the WebSocket client
+  implementation for subprotocol negotiation.
+* If the user application requests to be pinged (see [RFC 6455 section
+  5.5.2](https://tools.ietf.org/html/rfc6455#section-5.5.2)) in a
+  specific interval, the client SHALL set the field *ping_interval* to
+  the requested interval in seconds. Otherwise, *ping_interval* MUST be
+  set to `0` indicating that no WebSocket *ping* messages SHOULD be
+  sent.
+* If the client has stored the server's public permanent key (32 bytes)
+  and wants to verify it, it MUST include it in the *key* field.
```

Comment:
> How about renaming key to server_key.

I tried to follow the example set by other message types, where the key field is always just called `key`. E.g. in the "key" message, the field is not "session_key". (Although that wouldn't be too bad of an idea, but increases data transmitted.)